test/Api.Test/Tools/Controllers/ImportCiphersControllerTests.cs

```diff
@@ -92,6 +93,38 @@ await sutProvider.GetDependency<IImportCiphersCommand>()
             );
     }
 
+    [Theory, BitAutoData]
+    public async Task PostImportIndividual_WithArchivedDate_SavesArchivedDate(User user,
+        IFixture fixture, SutProvider<ImportCiphersController> sutProvider)
+    {
+        var archivedDate = DateTime.UtcNow;
+        sutProvider.GetDependency<GlobalSettings>()
+            .SelfHosted = false;
+
+        sutProvider.GetDependency<Core.Services.IUserService>()
+            .GetProperUserId(Arg.Any<ClaimsPrincipal>())
+            .Returns(user.Id);
+
+        var request = fixture.Build<ImportCiphersRequestModel>()
+            .With(x => x.Ciphers, fixture.Build<CipherRequestModel>()
+                .With(c => c.ArchivedDate, archivedDate)
+                .With(c => c.OrganizationId, Guid.NewGuid().ToString())
+                .With(c => c.FolderId, Guid.NewGuid().ToString())
```

Comment:
⛏️ The cipher when imported into an individual vault, won't have an orgId set and for the sake of the test a folderId is also not required.